src/main/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessage.java

```diff
@@ -248,6 +255,7 @@ public MessageBuilder(Builder builder) {
 
         private final Builder builder;
         private final Message message = new Message();
+        private  WindowsBuilder windowsBuilder;
```

Comment:
Because of this method https://github.com/aerogear/aerogear-unifiedpush-java-client/pull/68/files#diff-63e90d62fb5413ab881ba56b0eb54a32R373 , so we only initialize the windowsBuilder if needed but I can change that, make it final and always initialize it.